modules/t/dnaFragAdaptor.t

```diff
@@ -104,28 +104,6 @@ subtest "Test Bio::EnsEMBL::Compara::DBSQL::DnaFragAdaptor::fetch_by_GenomeDB_an
     done_testing();
 };
 
-subtest "Test Bio::EnsEMBL::Compara::DBSQL::DnaFragAdaptor::fetch_all_by_GenomeDB_region method", sub {
```

Comment:
Actually, the test itself has already been adapted to use `fetch_all_by_GenomeDB`. It's only the title of the subtest that needs change, but the test can stay